app/views/records/index.html.erb

```diff
@@ -1,23 +1,29 @@
+<%= render 'shared/navbar', title: "#{@records.name}", back_link: groups_path %>
+
 <div class = "transaction_group">
   <h2><%= @group.name %></h2>
   <h2>$<%= @group.total_amount %></h2>
 </div>
 
 <div class = "transactions_container">
-  <% @actions.each do |action| %>
+  <% @records.each do |record| %>
     <div class = "transaction_details">
       <div class = "transaction">
       <img src="https://png.pngtree.com/png-vector/20190130/ourlarge/pngtree-various-goods-and-cars-png-image_643136.jpg" />
-        <h2><%= action.name %></h2>
-        <h3>$<%= action.amount %></h3>
+        <h2><%= record.name %></h2>
+        <h3>$<%= record.amount %></h3>
       </div>
-      <p><%= "#{action.created_at.day} #{action.created_at.strftime("%b")} #{action.created_at.year}" %></p>
+      <p><%= "#{record.created_at.day} #{record.created_at.strftime("%b")} #{record.created_at.year}" %></p>
     </div>
   <% end %>
 
-  <% if @actions.empty? %>
+  <% if @records.empty? %>
     <div class="no_content">
       <p>No transactions available yet!</p>
     </div>
   <% end %>
+</div>
+
+<div class="group_btn">
+    <%= link_to 'Add New Transaction', new_group_record_path(@group), class: 'add-btn' %>
 </div>
```

Comment:
- [ ] as part of the requirements kindly make sure to follow exactly these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

- Colors
- Typography: font face, size and weight.
- Layout: composition and space between elements.

Your Design           | Required Design
:-------------------------:|:-------------------------:
![image](https://github.com/mnaibei/budget_app/assets/89282221/96e44d0f-8d4e-4287-a6ca-d03c235e4b1a) | ![image](https://github.com/mnaibei/budget_app/assets/89282221/ac8da16e-fa08-424e-a5c1-a9ef9f7e926c)


Your Design           | Required Design
:-------------------------:|:-------------------------:
![image](https://github.com/mnaibei/budget_app/assets/89282221/320cd680-a4c7-4041-a4ab-47dd02be5703) | ![image](https://github.com/mnaibei/budget_app/assets/89282221/8c25da9b-53e7-4ac0-9e36-40c5f161feea)

Your Design           | Required Design
:-------------------------:|:-------------------------:
![image](https://github.com/mnaibei/budget_app/assets/89282221/dc977e20-4507-4654-8aa5-11feb81a42c4) | ![image](https://github.com/mnaibei/budget_app/assets/89282221/c3939b74-5f6b-4159-9e6e-206f59f87623)

Your Design           | Required Design
:-------------------------:|:-------------------------:
![image](https://github.com/mnaibei/budget_app/assets/89282221/d2c75cd6-ae80-4334-a7e9-a58b158efe90) | ![image](https://github.com/mnaibei/budget_app/assets/89282221/7f2e65cc-cac6-4bad-896c-8e032f2d60a5)